packages/client/components/TeamPrompt/TeamPromptDiscussionDrawer.tsx

```diff
@@ -104,30 +142,51 @@ const TeamPromptDiscussionDrawer = ({meetingRef, isDesktop}: Props) => {
     })
   }
 
-  return localDiscussionId ? (
+  // :TODO: (jmtaber129): Use content from 'stage.stage'.
```

Comment:
+1 what's the reason for not using the response from the actual gql query?

Reply:
The resolver for the stage's `response` is stubbed on the backend for this branch's base (returns an empty object), so felt better to leave this as a stub on the frontend, and either implement it in a follow-up PR, or fix this before merging if https://github.com/ParabolInc/parabol/pull/6388 lands first

Reply:
makes sense, thank you!